contrib/test.sh

```diff
@@ -43,6 +43,23 @@ then
     )
 fi
 
+# Miri Checks if told to
+# Only supported in nightly
+if [ "$DO_MIRI" = true ]
+then
+    (
+        MIRI_NIGHTLY=nightly-$(curl -s https://rust-lang.github.io/rustup-components-history/x86_64-unknown-linux-gnu/miri)
+        echo "Installing latest nightly with Miri: $MIRI_NIGHTLY"
+        rustup set profile minimal
+        rustup default "$MIRI_NIGHTLY"
+        rustup component add miri
+        cargo miri test -- -- any_unsafe_transmute_miri_test
```

Comment:
This line filters to only run the `any_unsafe_transmute_miri_test` test? Maybe we should shorten it to `miri_`, so that in future, we can add MIRI-executable tests just by prefixing them with `miri_`